source/stylesheets/refills/_modal.scss

```diff
@@ -130,7 +130,7 @@
     }
 
     a.cta {
-      color: white;
+      color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.